tests/test_types.py

```diff
@@ -78,12 +79,29 @@ class DemoType(Record):
         self.assertIsInstance(demo.num, six.integer_types)
         demo.num = "123.0"
         self.assertIsInstance(demo.num, float)
+        demo.num = "nan"
+        self.assertIsInstance(demo.num, float)
 
         with self.assertRaises(exc.CoerceError):
             demo.num = "123.0a"
 
-        demo.num = "nan"
-        self.assertIsInstance(demo.num, float)
+        demo.num = decimal.Decimal("nan")
```

Comment:
These tests seem to be testing the decimal library which is already tested by the creators. Are you sure this is useful to have in here? It seems like it would make more sense to test the `to_json()` method with `Decimal` types instead of testing that `decimal.Decimal` works.

Reply:
I agree with this.  We should be testing the business logic behind the unit of work that we control.  Sorry I missed that.